### PR TITLE
chore: remove prints

### PR DESCRIPTION
**Pull request by enitrat**
Removes print statements, as they're quite gas consuming and they make some EF-tests reach OOG 😢

- - -
This change is [Reviewable](https://reviewable.io/reviews/kkrt-labs/kakarot-ssj/693)

### CODE REVIEW

File: crates/contracts/src/kakarot_core/kakarot.cairo [hunk 1]
```diff
@@ -504,7 +500,7 @@ mod KakarotCore {
             //     ) {
             //     Result::Ok(_) => {},
             //     Result::Err(err) => {
-            //         println!("process_transaction: sequencer couldn't charge gas");
+            //
```
Comment (bajpai244) [inline crates/contracts/src/kakarot_core/kakarot.cairo:503]: do we need comment here? I guess we can remove the whole line itself.

File: crates/evm/src/tests/test_state.cairo [hunk 2]
```diff
@@ -20,7 +20,7 @@ fn test_compute_state_key() {
     //         ];
     //     let values_to_hash = [keys[0],keys[1],keys[2]];
     //     let hash = poseidon_hash_many(&values_to_hash);
-    //     println!("hash: {}", hash);
+    //
```
Comment (bajpai244) [inline crates/evm/src/tests/test_state.cairo:23]: same, do we need `//` here? we can remove the whole line I guess.

File: crates/contracts/src/kakarot_core/kakarot.cairo (hunk 1 above)
Comment (enitrat) [reply crates/contracts/src/kakarot_core/kakarot.cairo:503]: yes, we need, it's explained above

File: crates/evm/src/tests/test_state.cairo (hunk 2 above)
Comment (enitrat) [reply crates/evm/src/tests/test_state.cairo:23]: (also for these kind of nitpicks it's easier if you make a suggestion so that I can do it in github)

**Review by bajpai244 — APPROVED** [review]

**Pull request merged**